sw/device/tests/crypto/cryptotest/firmware/sha3_sca.c

```diff
@@ -367,20 +400,34 @@ status_t handle_sha3_sca_disable_masking(ujson_t *uj) {
  * @param msg_len Message length.
  */
 sha3_sca_error_t sha3_serial_absorb(const uint8_t *msg, size_t msg_len) {
-  // Start a new message and write data to message FIFO.
+  // Start a new message.
   if (sha3_msg_start(kDifKmacModeSha3Len256) != kDifOk) {
     return sha3ScaAborted;
   }
+
+  if (fpga_mode == false) {
+    // Start command. On the chip, we need to first issue a START command
+    // before writing to the message FIFO.
+    sca_call_and_sleep(kmac_start_cmd, kIbexLoadHashPrefixKeySleepCycles);
```

Comment:
I think you don't need the `sca_call_and_sleep()` here. Because if running in SHA3 mode, there is no prefix and no key being loaded and hashed. Ibex doesn't need to wait. But I might be wrong.

Reply:
Fixed, thanks!